packages/foreman/foreman/foreman.spec

```
@@ -1005,18 +1005,20 @@ fi
 exit 0
 
 %post service
-%systemd_post %{name}.socket
 %systemd_post %{name}.service
 
 %preun service
-%systemd_preun %{name}.socket
 %systemd_preun %{name}.service
+%systemd_preun %{name}.socket
 
 %postun service
-%systemd_postun_with_restart %{name}.socket
 %systemd_postun_with_restart %{name}.service
+%systemd_postun_with_restart %{name}.socket
```

Did you intend this? I think you meant
```suggestion
%systemd_postun %{name}.socket
```